### PR TITLE
Atomics load add

### PR DESCRIPTION
**Pull request by rwalczyna**
Parallel tests:
Before: +1475 | -1005
After : +1516 | -964

#63 Should be merged first.

### CODE REVIEW

**Comment by rerobika** [comment]
Please rebase.

**Review by rerobika — CHANGES_REQUESTED** [review]
Only one request. Can you separate these operations into an individual source file?

**Comment by rerobika** [comment]
Also please keep in mind during rebasing that 3d43aa82e826ba6cfb787f412f88de55a6b30182 moved the `deps/jerry/jerryscript` directory into a submodule.

**Comment by rwalczyna** [comment]
I rebased the commit and moved Atomics methods to jerry_atomics.
I also moved functions used to register handlers (jerryx_handler_register_*) to utils - to use them outside of jerry_isolate.

'parallel' suite before (3d43aa82e826ba6cfb787f412f88de55a6b30182):
[14:30|% 100|+ 1588|- 892]: Done
after:
[12:38|% 100|+ 1640|- 840]: Done

File: deps/jerry/v8jerry/v8jerry_atomics.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        (const jerry_char_t*)"add", prototype_val, atomics_add_handler);
+    // Atomics.load
+    JerryxHandlerRegister(
+        (const jerry_char_t*)"load", prototype_val, atomics_load_handler);
+
+    jerry_release_value(prototype_val);
+    return;
+}
+
+}  // namespace JerryAtomics
```
Comment (rerobika) [inline deps/jerry/v8jerry/v8jerry_atomics.cpp]: Please add new line at the end of the file.

File: deps/jerry/v8jerry/v8jerry_utils.cpp [hunk 2]
```diff
@@ -150,3 +150,49 @@ void JerryForceCleanup(void) {
...
+    jerry_release_value (global_obj_val);
+
+    name_val = jerry_create_string((const jerry_char_t *) "prototype");
+    jerry_value_t prototype_val = jerry_get_property(string_val, name_val);
+    jerry_release_value (name_val);
+    jerry_release_value (string_val);
+
+    JerryxHandlerRegister (name_p, prototype_val, handler_p);
+    jerry_release_value (prototype_val);
+}
```
Comment (rerobika) [inline deps/jerry/v8jerry/v8jerry_utils.cpp]: Ditto.

**Review by rerobika — COMMENTED** [review]
LGTM, after these minor changes.

**Review by rerobika — APPROVED** [review]
LGTM

**Pull request merged**